src/pages/index.js

```diff
@@ -92,17 +90,11 @@ const DescriptiveText = styled(P)`
     margin-bottom: ${size.medium};
 `;
 
-export default ({ pageContext: { featuredArticles } }) => {
-    const { stream, videos } = useTwitchApi();
-    const { title } = useSiteMetadata();
-    const twitchVideo = useMemo(() => {
-        if (stream) return stream;
-        if (videos && videos.length) return videos[0];
-        return null;
-    }, [stream, videos]);
-
+const IndexPageContent = ({ stream, title, twitchVideo, featuredArticles }) => {
+    const theme = useTheme();
```

Comment:
Nice! Now we can use the hook. Not sure if there are any concerns with this pattern? (@greggb )